client/components/command-palette/index.tsx

```diff
@@ -79,7 +79,7 @@ export function CommandMenuGroup( {
 	return (
 		<Command.Group about="WPCOM">
 			{ commands.map( ( command ) => {
-				const itemValue = command.searchLabel ?? command.label;
+				const itemValue = [ command.label, command.searchLabel ].filter( Boolean ).join( '||' );
```

Comment:
Can we make the "magic separator" a constant in the new file? So we make sure the string will be same in this line and inside the filter.

Reply:
Sure, to confirm, you want it in a completely new file?